WordPress/src/main/res/layout/reader_interest_card.xml

```diff
@@ -22,7 +22,7 @@
             app:layout_constraintTop_toBottomOf="@id/interests_you_may_like_header" />
 
         <org.wordpress.android.widgets.WPTextView
-            style="@style/ReaderTextView.Interests.Header"
+            style="@style/ReaderTextView.Card.Header"
```

Comment:
The `recommended_blogs' card header should match the style of the `interests` card so I decided to reuse the style. I renamed it so it is a bit more clear.